rainier-core/BUILD

```diff
@@ -8,14 +8,12 @@ load(
 scala_library(
   name = 'rainierCore',
   deps = [
-    '//.rainier-shaded-asm:asmShaded',
```

Comment:
The changes to `-core` seem most notable to me. Does this look right?

Reply:
It looks like this is because we switched to treating the shaded jar as an "external" dep. If `bazel build ...` works, it's fine.